templates/base.html

```diff
@@ -3,24 +3,23 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
+    <script async type="text/javascript" src="../static/scripts/progressbar.js"></script>
     {{ range .Styles }}
     <link rel="stylesheet" href="{{.}}">
     {{ end }}
-
     {{  range .Scripts }}
     <script defer type="text/javascript" src="{{.}}"></script>
     {{ end }}
-    
     <title>{{ .Title }}</title>
+    <div id="loader"></div>
```

Comment:
chu think about this hacky line ?)

Reply:
Have you added this to simplify interacting with DOM in JS? Why is it hacky tho?